src/guide/views/button/ButtonExample.jsx

```diff
@@ -79,6 +80,17 @@ export default class ButtonExample extends Component {
           </PrimaryButton>
         </Example>
 
+        <Example title="AlertButton">
+          <Text>
+            This button lets the user perform a special action, like
+            &quot;Cancel&quot;. It&apos;s higher in the UI hierarchy than the
```

Comment:
I would say Cancel is not hard enough for an alert button. Maybe there are cases where you would use this button but in general Cancel would use a more neutral button because you wouldn't change anything,

I would suggest here to go with something like "Delete" where you have to be aware of the danger that is happening when you click that button.